bindings/csharp/ckzg.c

```diff
@@ -24,29 +26,17 @@ void free_trusted_setup_wrap(KZGSettings *s) {
   free(s);
 }
 
-void blob_to_kzg_commitment_wrap(uint8_t out[48], const Blob blob, const KZGSettings *s) {
+C_KZG_RET blob_to_kzg_commitment_wrap(uint8_t out[48], const Blob blob, const KZGSettings *s) {
```

Comment:
You need to put the result in `out` in this function - in the proposed changes `out` has become ignored.